index.js

```diff
@@ -771,6 +774,11 @@
     return x instanceof Right && R.eqProps('value', x, this);
   });
 
+  //  Right#extend :: Either a b ~> (b -> b) -> Either a b
+  Right.prototype.extend = def('Right#extend', [Function], function(f) {
+    return Right(f(this.value));
```

Comment:
We're currently inconsistent in our use of `new`. We use it in the (identical) definition of `Right#map` below. We should open a separate pull request to remove the unnecessary uses of `new`.